lib/fedora_migrate/rels_ext_datastream_mover.rb

```diff
@@ -35,23 +41,20 @@ def migrate_object(fc3_uri)
       RDF::URI.new(ActiveFedora::Base.id_to_uri(id_component(fc3_uri)))
     end
 
-    def id_component(uri)
-      uri.to_s.split(/:/).last
+    def is_missing?(uri)
+      return false if ActiveFedora::Base.exists?(id_component(uri))
+      Logger.warn "#{source.pid} could not migrate relationship to #{uri.to_s} because it doesn't exist in Fedora4"
```

Comment:
Could not migrate a particular relationship or any relationship?  If the former, it'd be good to specify which one. If the latter, maybe pluralize 'relationship.'

Minor niggly thing: s/Fedora4/Fedora 4/